socrates/lib/members/views/userinfoform.pug

```diff
@@ -29,6 +29,9 @@ mixin memberinfoform(member, subscriber, newmember_text, allCountries)
           +text('lastname', 'Last name', member.lastname())
         .col-sm-12
           +singleselect('country', 'Country of Residence', subscriber ? subscriber.country() : null, 'Select your country', allCountries)
+        .col-sm-12
+          +text('pronoun', 'Preferred pronoun (will appear on your name tag)', subscriber && (typeof subscriber.addon === 'function') ? subscriber.addon().pronoun():'')
```

Comment:
Das Subscriber Objekt hat eine addon() Methode, die als Lazy Initializer funktioniert. Die ruft das pug auf. Leider benutzt einer der bestehenden Integrationstests scheinbar ein Mock, dass dieses Verhalten nicht repliziert - und der Methodenaufruf produziert einen "addon is not a function" error. Ich habe mir heute den halben Tag die Finger fusselig gesucht, aber durch die string-basierte DI und die Tatsache, dass mocha nicht einmal den Namen des Tests ausgibt, habe ich die Wurzel des Übels nicht finden können. Darum habe ich ins pug den Guard eingebaut - damit läuft zumindest alles wieder so, wie erwartet.